lib/RecursiveDirectoryIteratorFilter.php

```diff
@@ -22,21 +22,18 @@
 
 namespace NC\Updater;
 
-class RecursiveDirectoryIteratorWithoutData extends \RecursiveFilterIterator {
-	public function accept(): bool {
-		$excludes = [
-			'.rnd',
-			'.well-known',
-			'data',
-			'..',
-		];
+class RecursiveDirectoryIteratorFilter extends \RecursiveFilterIterator {
+	private array $excludedPaths;
 
-		/** @var \SplFileInfo|false */
-		$current = $this->current();
-		if (!$current) {
-			return false;
-		}
+	public function __construct(
+		\RecursiveDirectoryIterator $iterator,
+		array $excludedPaths = ['data'],
+	) {
+		parent::__construct($iterator);
+		$this->excludedPaths = array_flip($excludedPaths);
+	}
 
-		return !(in_array($current->getFilename(), $excludes, true) || $current->isDir());
+	public function accept(): bool {
+		return !isset($this->excludedPaths[$this->current()->getFilename()]);
 	}
```

Comment:
previous version double-checked that current() does not return false, maybe it would be safer to do the same.
PHP documentation for the parent method is `mixed` return type :shrug: